pkg/api/types.go

```diff
@@ -138,6 +138,21 @@ type ObjectMeta struct {
 	// objects.  Annotation keys have the same formatting restrictions as Label keys. See the
 	// comments on Labels for details.
 	Annotations map[string]string `json:"annotations,omitempty"`
+
+	// List of objects depended by this object. If ALL objects in the list have
+	// been deleted, this object will be garbage collected.
+	OwnerReferences []ObjectReference `json:"ownerReferences,omitempty"`
```

Comment:
Here's @bgrant0607's reason for naming it "owner": https://github.com/kubernetes/kubernetes/pull/23656#discussion_r58469784